--- a/make/autoconf/lib-tests.m4
+++ b/make/autoconf/lib-tests.m4
@@ -64,16 +64,17 @@ AC_DEFUN_ONCE([LIB_TESTS_SETUP_GTEST],
         # determined. :-( Instead, there are different, incorrect version
         # numbers we can look for.
         GTEST_VERSION_1="`$GREP GOOGLETEST_VERSION $GTEST_FRAMEWORK_SRC/CMakeLists.txt | $SED -E -e 's/set\(GOOGLETEST_VERSION (.*)\)/\1/'`"
-        if test "x$GTEST_VERSION_1" != "x1.9.0"; then
-          AC_MSG_ERROR([gtest at $GTEST_FRAMEWORK_SRC does not seem to be version 1.8.1])
+        if test "x$GTEST_VERSION_1" != "x1.14.0"; then
+          AC_MSG_ERROR([gtest at $GTEST_FRAMEWORK_SRC does not seem to be version 1.14.0])
         fi
 
+        # $GTEST_FRAMEWORK_SRC/configure.ac does not exist
         # We cannot grep for "AC_IN*T" as a literal since then m4 will treat it as a macro
         # and expand it.
         # Additional [] needed to keep m4 from mangling shell constructs.
         [ GTEST_VERSION_2="`$GREP -A1 ^.C_INIT $GTEST_FRAMEWORK_SRC/configure.ac | $TAIL -n 1 | $SED -E -e 's/ +\[(.*)],/\1/'`" ]
-        if test "x$GTEST_VERSION_2" != "x1.8.0"; then
-          AC_MSG_ERROR([gtest at $GTEST_FRAMEWORK_SRC does not seem to be version 1.8.1 B])
+        if false && test "x$GTEST_VERSION_2" != "x1.14.0"; then
+          AC_MSG_ERROR([gtest at $GTEST_FRAMEWORK_SRC does not seem to be version 1.14.0 B])
         fi
       fi
     fi
